pkgs/by-name/li/libvpl/package.nix

```diff
@@ -32,6 +32,8 @@ stdenv.mkDerivation (finalAttrs: {
       "-DBUILD_TOOLS=OFF"
   ];
 
+  patches = [ ./opengl-driver-lib.patch ];
```

Comment:
Please bring this patch upstream. I would not expect them to want our NixOS specific path in there by default but this could be the starting point for adding a generic mechanism with configurable defaults that we'd then use.

Reply:
(And then add a back link as a comment.)